src/index.ts

```diff
@@ -77,21 +86,26 @@ client.on(Events.InteractionCreate, async interaction => {
     try {
         if (commandName === START_CONTEST_COMMAND) {
             contestManager.startContest();
-            await interaction.reply('El concurso ha comenzado!');
+            await interaction.reply('¡El concurso ha comenzado!');
         } else if (commandName === WINNER_COMMAND) {
             const winners = await getSortedLeaderboard(3, 'meme');
             const bones = await getSortedLeaderboard(3, 'bone');
-            if (winners.length == 0 && bones.length == 0) {
+            const medals = await getSortedLeaderboard(3, 'medal');
+
+            if (winners.length === 0 && bones.length === 0) {
                 await interaction.reply('No winners found for this week.');
```

Comment:
Wonder why do we use English on here

Reply:
Fixed, now all the logs are in Spanish, thanks Yeiner!